web/src/beta/features/Editor/Map/LayerStylePanel/Editor/StyleInterface/convert.ts

```
@@ -0,0 +1,140 @@
+};
+
+export const converToLayerStyleValue = (
+  styleNodes: StyleNodes
+): Partial<LayerAppearanceTypes> => {
+  return appearanceTypes.reduce((acc, cur) => {
+    return styleNodes[cur].length > 0
+      ? {
+          ...acc,
+          [cur]: styleNodes[cur].reduce((acc2, cur2) => {
+            return {
+              ...acc2,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)